mockredis/client.py

```diff
@@ -1354,6 +1356,27 @@ def _normalize_command_response(self, command, response):
 
         return response
 
+    # Config Set/Get commands #
+
+    def config_set(self, name, value):
+        """
+        Set a configuration parameter.
+        """
+        self.redis_config[name] = value
+
+    def config_get(self, pattern='*'):
+        """
+        Get one or more configuration parameters.
+        """
+        result = {}
+        for name, value in self.redis_config.items():
+            if fnmatch.fnmatch(name, pattern):
+                try:
+                    result[name] = int(value)
```

Comment:
Sorry was just looking at the documentation and ran a quick test observed that 
`
>>> r.config_get('*max-*-entries*')
{'zset-max-ziplist-entries': '128', 'hash-max-ziplist-entries': '512', 'set-max-intset-entries': '512'}
`
So basically you don't need to convert them into int.